kube/deploy.sh

```diff
@@ -1,11 +1,12 @@
 #!/bin/bash
 
-[ -z $4 ] && echo "Usage: $0 <namespace> <OpenVPN URL> <service cidr> <pod cidr>" && exit 1
+[ -z $5 ] && echo "Usage: $0 <namespace> <OpenVPN URL> <service cidr> <pod cidr> <cluster domain>" && exit 1
 
 namespace=$1
 serverurl=$2
 servicecidr=$3
 podcidr=$4
+domain=$5
```

Comment:
To be friendly to new users we should set this to a sane default. Can you change it to:

```
domain=${5:-svc.cluster.local}
```